README.md

```diff
@@ -62,7 +62,7 @@
       - [Speckle Analysis](#speckle_analysis)
       - [Interferometry or Holography](#interferometry_or_holography)
       - [Wavefront sensing](#wavefront_sensing)
-
+      - [Super-resolution imaging](#super_resolution_imaging)
```

Comment:
there is another subsection with a very similar name ("Super-resolution", under "Reconstruction of Visual Information"). Could we try to rename to tell them apart? This one seems to be about microscopy and super-resolution.

Reply:
Yes, I changed the former to "image super-resolution" and the latter to "optical super-resolution". What do you think?